package.json

```
@@ -85,17 +87,18 @@
     "remark-cli": "^9.0.0",
     "remark-preset-wooorm": "^8.0.0",
     "resolve-from": "^5.0.0",
+    "rollup": "^2.0.0",
+    "rollup-plugin-terser": "^7.0.0",
     "tape": "^5.0.0",
-    "tinyify": "^3.0.0",
     "xo": "^0.34.0"
   },
   "scripts": {
     "generate-constant-typings": "node script/generate-constant-typings",
     "generate-expressions": "node script/generate-expressions",
     "generate-dist": "babel lib --out-dir dist --quiet --retain-lines --plugins babel-plugin-unassert,./script/babel-transform-undebug.js,./script/babel-transform-constants.js; node script/copy-dict.js; prettier dist --loglevel error --write",
-    "generate-size": "browserify . -p tinyify -s micromark -o micromark.min.js; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
+    "generate-size": "rollup -c --silent; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
     "generate": "npm run generate-constant-typings && npm run generate-expressions && npm run generate-dist && npm run generate-size",
-    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "format": "remark . -qfo && prettier . --write --loglevel warn && xo --fix",
```

```suggestion
    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
```

haha, you really like the long form 😅 If you feel strongly, I’m fine with it. Otherwise, you can apply this suggestion

---

Ah, short form works with reinstalled node_modules. Probably was a bug in old prettier. But yeah, let's use long version for consistency :)

---

Uhm, the consistency is that short flags are used where possible, and it was that way before, hence why I’m suggesting to go back to how it was 😅

Anyway, feel free to commit this suggestion, or not. I think the rest looks good. Anything else?

---

In other script --write was used